gatsby-site/gatsby-config.js

```diff
@@ -1,7 +1,14 @@
 module.exports = {
   siteMetadata: {
-    title: `Support Documents`,
-    siteUrl: `https://support.sparkpost.com`,
+    siteName: `SparkPost Support Documentation`,
+    url: `https://support.sparkpost.com`,
+    title: `SparkPost Support Documentation`,
+    defaultTitle: "Default title",
```

Comment:
This is from prod? Let's give it something, at least. Maybe just the same as the title?

Reply:
Added some better values here, thanks!